README.md

```diff
@@ -1 +1,32 @@
-# Portfolio-Website
\ No newline at end of file
+# Portfolio-Website
+Portfolio: setup and mobile version skeleton
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
```

Comment:
please try to do the changes that the previous reviewer requested you by adding here a `## Getting started`  header like that and the details on how can a person run your project in their machines